docs/source/config/encoders/esjson.rst

```diff
@@ -36,7 +36,7 @@ Config:
     indexed into ElasticSearch. Available fields to choose are "Uuid",
     "Timestamp", "Type", "Logger", "Severity", "Payload", "EnvVersion", "Pid",
     "Hostname", and "Fields" (where "Fields" causes the inclusion of any and
```

Comment:
I think "Fields" should be changed to "DynamicFields", so there's a more clear link btn the value that is here and the `dynamic_fields` setting that it's connected to. I was originally thinking we could leave "Fields" in as a deprecated option for one release cycle, for b/w compatibility, but I'm not sure it's worth it...